timeseries/src/main/scala/com/criteo/cuttle/timeseries/TimeSeriesScheduler.scala

```diff
@@ -527,20 +535,32 @@ case class TimeSeriesScheduler(logger: Logger) extends Scheduler[TimeSeries] wit
     val parentsMap = workflow.edges.groupBy { case (child, _, _)   => child }
     val childrenMap = workflow.edges.groupBy { case (_, parent, _) => parent }
 
+    def reverseDescr(dep: TimeSeriesDependency) =
+      TimeSeriesDependency(dep.offsetHigh.negated, dep.offsetLow.negated)
+    def applyDep(dep: TimeSeriesDependency) = Function.unlift { (i: Interval[Instant]) =>
```

Comment:
it can be better to have the function return type... Not easy to understand without IDE...

Reply:
sure